### PR TITLE
Audient.EVO version 4.2.3.0

### PR DESCRIPTION
**Pull request by gr4vem1nd**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60122)

### CODE REVIEW

**Comment by gr4vem1nd** [comment]
> @gr4vem1nd,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.

All the official download links are from cloudfront.net :
https://evo.audio/products/evo-4/downloads/
https://evo.audio/products/evo-8/downloads/
https://support.audient.com/hc/en-us/articles/4403651019540

File: manifests/a/Audient/EVO/4.2.3.0/Audient.EVO.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+PackageIdentifier: Audient.EVO
+PackageVersion: 4.2.3.0
+PackageLocale: en-US
+Publisher: Audient Ltd
+PublisherUrl: https://audient.com
+PublisherSupportUrl: https://support.audient.com
+PrivacyUrl: https://www.iubenda.com/privacy-policy/76513121
+PackageName: EVO
+PackageUrl: https://evo.audio/
+License: missing
```
Comment (ItzLevvie) [inline manifests/a/Audient/EVO/4.2.3.0/Audient.EVO.locale.en-US.yaml:13]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**